client/blocks/plugins-scheduled-updates/schedule-list.tsx

```diff
@@ -117,6 +128,11 @@ export const ScheduleList = ( props: Props ) => {
 					{ schedules.length === 0 && isLoading && <Spinner /> }
 					{ ! isLoading && showScheduleListEmpty && (
 						<ScheduleListEmpty
+							pluginsUrl={
+								isGlobalSiteViewEnabled
+									? `${ siteAdminUrl }plugin-install.php`
+									: `/plugins/${ siteSlug }`
+							}
```

Comment:
Can we move this logic with logic for getting `isGlobalSiteViewEnabled` and `siteAdminUrl` to the hook and keep this component clean?

It can be the `usePluguinsUrl` hook.

What do you think?